recipes/filter-keystone-v2.rb

```diff
@@ -0,0 +1,35 @@
+#
+# Cookbook Name:: repose
+# Recipe:: filter-keystone-v2
+#
+# Copyright (C) 2013 Rackspace Hosting
+#
+
+include_recipe 'repose::install'
+
+unless node['repose']['filters'].include? 'keystone-v2'
+  filters = node['repose']['filters'] + ['keystone-v2']
+  node.normal['repose']['filters'] = filters
+end
+
+template "#{node['repose']['config_directory']}/keystone-v2.cfg.xml" do
+  owner node['repose']['owner']
+  group node['repose']['group']
+  mode '0644'
+  variables(
+    username: node['repose']['keystone_v2']['username_admin'],
+    password: node['repose']['keystone_v2']['password_admin'],
+    uri: node['repose']['keystone_v2']['uri'],
+    pool: node['repose']['keystone_v2']['pool'],
+    groups_in_header: node['repose']['keystone_v2']['groups_in_header'],
+    roles_in_header: node['repose']['keystone_v2']['roles_in_header'],
+    catalog_in_header: node['repose']['keystone_v2']['catalog_in_header'],
+    delegating_quality: node['repose']['keystone_v2']['delegating_quality'], 
+    white_list: node['repose']['keystone_v2']['white_list'],
+    cache: node['repose']['keystone_v2']['cache'],
+    tenant_handling: node['repose']['keystone_v2']['tenant_handling'],
+    require_service_endpoint: node['repose']['keystone_v2']['require_service_endpoint'],
+    pre_authorized_roles: node['repose']['keystone_v2']['pre_authorize_roles']
```

Comment:
should be `node['repose']['keystone_v2']['pre_authorized_roles']`, you missed a `d`.